symfony/phpunit-bridge/4.1/tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+(new Dotenv())->load(__DIR__.'/../.env', __DIR__.'/../.env.test');
```

Comment:
If an env var is specified in both, `.env.test` wins, right?

Also, if we do this for the `test` environment, people will ask for it for other environments too. And I think they're right - it would be simpler to have a file like this that will load from `.env.{ENV}` (if it exists) and to require that from `index.php` and `console`. `console` is especially important - we don't want someone to run a `bin/console` test with `--env=test` and have different environment variables than their tests.

Check out https://github.com/symfony/recipes/pull/408/files - where we working on something similar to this. We were also putting even some extra code into this central "bootstrap" file to fully remove the duplication between `index.php`, `console` (and now) the test bootstrap.